docs/Project.toml

```diff
@@ -1,4 +1,5 @@
 [deps]
+ControlSystemsBase = "aaaaaaaa-a6ca-5380-bf3e-84a91bcd477e"
```

Comment:
Remember compat.

Reply:
This is in `docs/Project.toml` where no packages have compat so far :P 
https://github.com/SciML/ModelingToolkitStandardLibrary.jl/blob/de1151f3381b6f4c9a3205c767a076d1627308e6/docs/Project.toml